### ISSUE TITLE
SQLite segmentation fault on TEXT field longer than 64 characters

### ISSUE THREAD

**Issue by enykeev** [oven-sh/bun#4228]
### What version of Bun is running?

0.7.2

### What platform is your computer?

Linux 5.15.90.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

```typescript
import { Database } from "bun:sqlite";

const db = new Database();

db.prepare(`
CREATE TABLE IF NOT EXISTS flags (
  id TEXT NOT NULL,
  description TEXT,
  PRIMARY KEY (id)
)
`).run();

db.prepare(
  "INSERT INTO flags (id, description) VALUES (?, ?)",
).run(
  "1",
  "a".repeat(64),
);

const res = db.prepare("SELECT * FROM flags where id = '1'").get();

console.log(res);

db.prepare(
  "INSERT INTO flags (id, description) VALUES (?, ?)",
).run(
  "2",
  "a".repeat(65),
);

const res2 = db.prepare("SELECT * FROM flags where id = '2'").get();

console.log(res2);
console.log(res2.description)
```

### What is the expected behavior?

```
$ bun run packages/core/checksqlite.ts
{
  id: "1",
  description: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
}
{
  id: "2",
  description: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
}
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

### What do you see instead?

```
$ bun run packages/core/checksqlite.ts
{
  id: "1",
  description: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
}
{
  id: "2"
}
[1]    3076859 segmentation fault  bun run packages/core/checksqlite.ts
```

**Comment by Jarred-Sumner** [comment]
This was fixed in v0.7.3. Please run `bun upgrade`

**Comment by enykeev** [comment]
Seems like it was a duplicate of #3991. Can confirm that upgrading worked.
